content/docs/release-notes/2305-0-release-notes.md

```diff
@@ -0,0 +1,202 @@
+* Added support for stereoscopic rendering (such as VR support) via OpenXR within O3DE. This support was added through the Vulkan backend and it has been extensively tested using Quest 2. As part of this effort we are able to support Link mode where the rendering is done on PC and can be viewed on a stereoscopic display or the app can be run on the device natively. A special optimized VR pipeline has been added to Atom to help support running the app natively on the device.
+* Expanded the texture streaming support to Vulkan backend. Now we have image streaming control which can load or evict texture mips automatically at runtime based on the streaming image pool's budget.
+* Updated the shader library, material system, and builders to support a layer of abstraction between lighting and materials, called the material pipeline. It uses new builders and new features from the asset system to build unique shaders for each render pipeline. Material types can provide custom data structures and evaluator functions for vertex, pixel, geometry, and lighting stages. The builder inserts any of the custom include files with overridden data types and functions as it builds all of the combinations of shaders listed in the material pipeline.  
+	* The result of this is a concrete material type referencing all of the different combinations of shaders for each pipeline, with the customization stitched in. All of this is saved to the intermediate assets folder and processed by the AP to generate Final product material types and shaders. The material pipeline also includes a Lua script that will be invoked whatever material properties or the active render pipeline changes to enable or disable shaders based on the current state.  
+	* All of the physically based rendering materials, as well as other shaders and material types included with Atom have been converted to the system.  
+	* All of the Atom Sample Viewer (ASV) samples and tests have also been updated to use this system. ASV also includes different experimental render and material pipelines to demonstrate how existing materials that use this system can be applied to a standard, deferred rendering pipeline.
+* Updated the temporal anti-aliasing (TAA) pass by splitting it into two passes, which allows the TAA's history buffer to be copied and separated from the TAA algorithm itself. This required a change to `PostProcessParent.pass` to reference `TaaParentTemplate` instead of `TaaTemplate`. If your project overrides `PostProcessParent.pass` and uses TAA, then it will need to be updated to point to the new TAA parent pass.
+* Updated `LuaIDE` so developers can now override it and use a different Lua Editor/Debugger. For more information, refer to [Working with an External Lua Debugger](https://github.com/o3de/o3de/wiki/Working-With-An-External-Lua-Debugger).
+
+
+### sig-network
```

Comment:
This section is missing updates the SIG made in https://github.com/o3de/sig-release/pull/173/files , it should be:


* Provides a major update to the Multiplayer sample game: https://github.com/o3de/o3de-multiplayersample to provide a feature rich game experience with UX, audio, visual FX, player vs player and player vs environment elements.
* Provides a new simple player spawner component that avoids the needs to write custom spawn logic, [#13871](https://github.com/o3de/o3de/pull/13871)
* Adds a validation mechanism to identify when client and server have differences in networked code and properties to aid debugging of these issues.
* Provides method for code separation between client and server side code to prevent any exposure of server side logic to clients. Provides a new Unified Launcher target to aid local testing.

Reply:
this is addressed now